autocertkit/network_tests.py

```diff
@@ -739,7 +739,7 @@ def _verify_ethtool_offloads(self, session, device):
                 # If it is not possible due to the hardware ristriction, then ACK should
                 # ignore this failure and keep running tests.
                 if '[fixed]' in hw_offloads[k]:
-                    raise FixedOffloadException("Required offload %s is fixed to %s." % (k, hw_offloads[k]))
+                    log.debug(FixedOffloadException("Required offload %s is fixed to %s." % (k, hw_offloads[k])))
```

Comment:
Apologies. It seems I forgot to push, the last commit. I just did. Sorry about that.